### ISSUE TITLE
Iterate on "block guide" placement and phrasing in block settings

### ISSUE THREAD

**Issue by annezazu** [Automattic/wp-calypso#95169]
### What

For some blocks, not all, there's a "Block guide" link in the settings section. This is both easy to miss and, in the case of the Query Loop block, is a bit awkward design wise. 

Example with image block:

![Image](https://github.com/user-attachments/assets/09c0fa40-dccd-4732-9526-79978a6bf150)

Example with Query Loop block:

![Image](https://github.com/user-attachments/assets/892d214d-0b0a-4831-a20c-942ea0c5fff4)

I think this could be improved in a few ways to help more folks find this when they need it and make the most of it: 
- Change "Block guide" phrasing to something a bit clearer, like "Learn more". I wasn't sure what a block guide would be when I first clicked on it.
- When you open the "Block guide" the panel covers the block settings, making it hard to follow the instructions along when it's covering what you need to change. 
- When you minimize the "Block guide" it resets placement, even if you clicked on a specific anchor/heading.
- Fix the spacing when another link is in place, like add new post for the Query Loop block. I wonder if we can find a way so that the "Block guide" is listed after the "Add new post" option.
- Ensure all core blocks have a link rather than just a few so it becomes a more predictable pathway to get help. 

### Why

To help folks make better use out of the block guides.

**Comment by richtabor** [comment]
Do we know if folks are engaging with this?

**Comment by annezazu** [comment]
@Automattic/vertex is this something you all can look into as part of the broader Help Center work you're doing?

**Comment by escapemanuele** [comment]
We can look at some of the issues!

> Ensure all core blocks have a link rather than just a few so it becomes a more predictable pathway to get help.

But I imagine this is not related to us?

**Comment by escapemanuele** [comment]
- https://github.com/Automattic/wp-calypso/issues/96630
- https://github.com/Automattic/dotcom-forge/issues/10075
- https://github.com/Automattic/dotcom-forge/issues/10076
